docs/tutorials/acme/dns-validation.rst

```diff
@@ -86,30 +75,45 @@ Once we have created the above Issuer we can use it to obtain a certificate.
 .. code-block:: yaml
    :linenos:
 
-   apiVersion: certmanager.k8s.io/v1alpha1
-   kind: Certificate
-   metadata:
-     name: example-com
-     namespace: default
-   spec:
-     secretName: example-com-tls
-     issuerRef:
-       name: letsencrypt-staging
-     commonName: '*.example.com'
-     dnsNames:
-     - example.com
-     - foo.com
-     acme:
-       config:
-       - dns01:
-           provider: prod-dns
-         domains:
-         - '*.example.com'
-         - example.com
-       - dns01:
-           provider: cf-dns
-         domains:
-         - foo.com
+    apiVersion: certmanager.k8s.io/v1alpha1
+    kind: Certificate
+    metadata:
+      creationTimestamp: 2019-08-30T04:06:43Z
+      generation: 1
+      labels:
+        app.kubernetes.io/instance: imprecise-dingo
+        app.kubernetes.io/managed-by: Tiller
+        app.kubernetes.io/name: controlcenter
+        app.kubernetes.io/version: "1.0"
+        helm.sh/chart: example-com-0.1.0
+      name: quickstart-example-tls
+      namespace: default
+      ownerReferences:
+      - apiVersion: extensions/v1beta1
+        blockOwnerDeletion: true
+        controller: true
+        kind: Ingress
+        name: example-com-0.1.0
+        uid: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+      resourceVersion: "2964052"
+      selfLink: /apis/certmanager.k8s.io/v1alpha1/namespaces/default/certificates/quickstart-example-tls
+      uid: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
```

Comment:
Can you strip out all the irrelevant metadata here so the resource's `metadata` looks similar to how it did before (as well as changing the name and dnsNames to align too)?

Reply:
ok. Let me do it on Monday

Reply:
added